frontend/web/components/Aside.js

```diff
@@ -77,6 +78,7 @@ const Aside = class extends Component {
               ? {}
               : { live_from_after: new Date().toISOString() },
           )
+          AppActions.getEnv(this.props.environmentId)
```

Comment:
I'm confused by this, we're already getting the project's environments in AppActions.getProject

Reply:
This is triggered every time we change environments. It is done this way because we need to save the response to the store so we can use its data to determine the content of the alert and whether it will be displayed.

Reply:
We use the environment endpoint because the project one doesn't have the limits information we need, and Matt asked to keep it this way since it will be more efficient in terms of querying the DB